Xamarin.Forms.Platform.iOS/Renderers/DatePickerRenderer.cs

```diff
@@ -167,27 +167,12 @@ void UpdateDateFromModel(bool animate)
 		{
 			if (_picker.Date.ToDateTime().Date != Element.Date.Date)
 				_picker.SetDate(Element.Date.ToNSDate(), animate);
-
-			// Can't use Element.Format because it won't display the correct format if the region and language are set differently
-			if (string.IsNullOrWhiteSpace(Element.Format) || Element.Format.Equals("d") || Element.Format.Equals("D"))
+						
+			if (string.IsNullOrWhiteSpace(Element.Format))
 			{
-				NSDateFormatter dateFormatter = new NSDateFormatter();
```

Comment:
In Android we are using the following code:

```
if (String.IsNullOrWhiteSpace(Element.Format))
{
    EditText.Text = date.ToShortDateString();
}
else if (Element.Format.Contains('/'))
{
    EditText.Text = date.ToString(Element.Format, CultureInfo.InvariantCulture);
}
else
{
    EditText.Text = date.ToString(Element.Format);
}
```

The default template is "d", and for that reason using `NSDateFormatter` with Short DateStyle the format is: "MM/dd/yy". In this PR we adapt iOS code to the other platforms.